### PR TITLE
Change Accent and Header Background Color for Fast Templates

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Addresses #2983

Looks like

FastGridTemplate
FastGridTemplateDark
FastListTemplate
FastListTemplateDark

### CODE REVIEW

File: panel/template/fast/base.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 class FastBaseTemplate(BasicTemplate):
 
-    accent_base_color = param.String(doc="""
+    accent_base_color = param.String(default="#0072B5", doc="""
```
Comment (MarcSkovMadsen) [inline panel/template/fast/base.py:15]: I wanted a way for the user to get the color for their app to make it easier to use the color in plots etc.

**Comment by philippjfr** [comment]
I like the color but really we need to be way more consistent about theming and styling. If we make this change it needs to apply to all templates and we should define a fixed color palette for both light (default) and dark themes which all the template specific themes can reference somehow.

**Comment by MarcSkovMadsen** [comment]
I don't think vanilla, bootstrap, material and fast templates can be alike. They are different design systems with very different properties.

And the Fast Templates are so far ahead of the rest of the templates. Who will ever take the time to develop the required css files for the other templates?

**Comment by philippjfr** [comment]
>I don't think vanilla, bootstrap, material and fast templates can be alike. They are different design systems with very different properties.

I'm not saying the need to be alike but I think there should a central definition of the default color palette, which is then reused by most of the themes. 

```
default_palette = {
    'primary': ...,
    'accent': ...,
    'secondary': ...,
    ...
}
```

and then we can have a templated CSS file like this:

```
:root {
  --pn-primary-color: {{ primary }};
  --pn-accent-color: {{ accent }};
  --pn-secondary-color: {{ secondary }};
}
```

then finally each template can use those variables to control its color scheme, e.g. the fast template CSS can then just use: 

```
:root {
    --container-background: var(--pn-primary-color);
}
```

**Comment by MarcSkovMadsen** [comment]
I'm not quite sure what the status on this is. Would it be possible to get into 0.13? Is there anything I should do to get this in?

(It would make my "developer advocate" work much easier to have this in. Then I would not have to always pick some more beautiful colors to make examples look nice. And the new users would have less code to worry about).

**Comment by MarcSkovMadsen** [comment]
> > I don't think vanilla, bootstrap, material and fast templates can be alike. They are different design systems with very different properties.
> 
> I'm not saying the need to be alike but I think there should a central definition of the default color palette, which is then reused by most of the templates.
> 
> ```
> default_palette = {
>     'primary': ...,
>     'accent': ...,
>     'secondary': ...,
>     ...
> }
> ```
> 
> and then we can have a templated CSS file like this:
> 
> ```
> :root {
>   --pn-primary-color: {{ primary }};
>   --pn-accent-color: {{ accent }};
>   --pn-secondary-color: {{ secondary }};
> }
> ```
> 
> then finally each template can use those variables to control its color scheme, e.g. the fast template CSS can then just use:
> 
> ```
> :root {
>     --container-background: var(--pn-primary-color);
> }
> ```

This problem here is that the `accent color` in Fast corresponds to the `accent` color in Bootstrap. We need some decision on whether to only use `primary` color terminology only. Or we use different terminologies for different frameworks, so that it is easier to relate to their documentation. In my head I've been implementing, describing everything as the latter.

**Comment by MarcSkovMadsen** [comment]
If you don't like BLUE, we might be able to find a better green than the suggested one :-)

**Comment by philippjfr** [comment]
Will merge and probably follow up separately.

**Pull request merged**